backend/routes.js

```diff
@@ -266,20 +266,22 @@ export default class Servers {
                         existingUsers.push({username, accessToken: access_token});
                         status = 201;
                     }
-                    saveSetting('USERS', existingUsers);
 
                     if (contains(username, getSetting('ALLOWED_USERS'))) {
-                        res.setCookie('plotly-auth-token', access_token, {'path': '/'});
+                        res.setCookie('plotly-auth-token', access_token, {'path': getSetting('DB_CONNECTOR_URL')});
```

Comment:
it looks like this is missing `secure`. We should probably keep this dry and make these options a constant like
```
COOKIE_OPTIONS = {
    'path': ...,
    'secure': '...
}
```